runtime/vm/JFRChunkWriter.hpp

```diff
@@ -600,6 +604,31 @@ class VM_JFRChunkWriter {
 		_bufferWriter->writeLEB128PaddedU32(dataStart, _bufferWriter->getCursor() - dataStart);
 	}
 
+	static void
```

Comment:
move this to the cpp file

Reply:
Why do we want that? Did we decide to move all the write functions to the cpp file?

Reply:
Yes, we should move all the write to .cpp. I made a mistake with the initial set. For now just move the new method, you can move the rest in a separate PR.